rllib/models/torch/encoders/vector.py

```diff
@@ -0,0 +1,129 @@
+from typing import TYPE_CHECKING, List
+
+import torch
+from torch import nn
+
+from ray.rllib.models.specs.specs_dict import ModelSpec
+from ray.rllib.models.torch.model import TorchModel
+from ray.rllib.models.utils import get_activation_fn
+from ray.rllib.utils.nested_dict import NestedDict
+
+if TYPE_CHECKING:
+    from ray.rllib.models.configs.encoder import VectorEncoderConfig
+
+
+def input_to_output_spec(
+    input_spec: ModelSpec, output_key: str, feature_dims: List[int]
+) -> ModelSpec:
+    """Convert an input spec to an output spec by replacing the input dims and
+    key with the output dims and output key.
+
+    E.g.
+    input_to_output_spec(
+        input_spec=ModelSpec({
+            "bork": "a, b, c, d",
+            "dork": "e, f, g, h"
+            }, h=2, d=3
+        ),
+        output_key="foo",
+        feature_dims=(2,)
+    )
+
+    should return:
+    ModelSpec({"foo": "a, b, c, d"}, d=2)
```

Comment:
Also, can you give another example, where we have >1 feature dims?

Reply:
Rewrote this function to be more clear